pom.xml

```diff
@@ -142,7 +142,7 @@
         <dependency>
```

Comment:
Why are these changes in here? 
I suggest not doing 'manual' updates but start using the renovate PRs

Reply:
there were so many renovate PR's. So I thought to have them in a single commit.
Agree It's good to handle them from PR's. Now onwards I will use renovate PR's